server/api/tumdev/campus_backend.proto

```diff
@@ -13,9 +13,22 @@ import "google/protobuf/empty.proto";
 import "google/protobuf/timestamp.proto";
 
 service Campus {
-  rpc GetTopNews (google.protobuf.Empty) returns (GetTopNewsReply) {
+  rpc GetTopNewsAlert (google.protobuf.Empty) returns (GetTopNewsAlertReply) {
     option (google.api.http) = {
-      get: "/news/top"
+      get: "/news/alerts/top"
+      response_body: "alert"
+    };
+  }
+  rpc GetNewsAlert (GetNewsAlertRequest) returns (GetNewsAlertReply) {
+    option (google.api.http) = {
+      get: "/news/alerts/{id}"
+      response_body: "alert"
+    };
+  }
+  rpc GetNewsAlerts (google.protobuf.Empty) returns (GetNewsAlertsReply) {
```

Comment:
Try to avoid `Empty` as changing it to an actual Request payload is not compatible on the wire format and might cause incompatibility if we want to extend this later on.

> Because field types are also used to determine when the receiver ends a field, changing field types across versions without carefully checking for compatibility can also cause problems. There are many specific [rules](https://developers.google.com/protocol-buffers/docs/proto3#updating) about how to change field types, but a good standard is to avoid ever changing the wire type of any field. If you’re in a situation where you need to change a field type, the best path forward is to deprecate the existing field and put the new information into a new field.

https://earthly.dev/blog/backward-and-forward-compatibility/